community/membership.md

```diff
@@ -260,6 +260,10 @@ with no significant contributions within the last 12 months. Contributions are
 measured using the CNCF [DevStats project], GitHub activity history, and other media
 by which a history of contributions can be reliably determined.
 
+Inactive reviewers or approvers are defined as members of one of the Argoproj
+Organizations who have not reviewed any pull requests in their sub-project's main
```

Comment:
or created any PRs -- since we consider both creating and reviewing for Reviewer+ roles